inc/compatibilities/shareaholic.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+/**
+ * Class Optml_sassy_social_share.
+ *
+ * @reason Shareaholic picks eco images to share
+ */
+class Optml_shareaholic extends Optml_compatibility {
+	/**
+	 * Should we load the integration logic.
+	 *
+	 * @return bool Should we load.
+	 */
+	function should_load() {
+		include_once( ABSPATH . 'wp-admin/includes/plugin.php' );
+
+		if ( ! is_plugin_active( 'shareaholic/shareaholic.php' ) ) {
+			return false;
+		}
+		return true;
+	}
+
+	/**
+	 * Register integration details.
+	 */
+	public function register() {
+		add_action(
+			'wp_enqueue_scripts',
+			function () {
+				wp_register_script( 'optml-typekit', false );
+				wp_enqueue_script( 'optml-typekit' );
```

Comment:
you use the same handle as [here](https://github.com/Codeinwp/optimole-wp/blob/master/inc/compatibilities/sassy_social_share.php#L49), maybe we can use something unique